src/py42/_internal/clients/archive.py

```diff
@@ -10,6 +11,76 @@ def get_data_key_token(self, device_guid):
         data = {u"computerGuid": device_guid}
         return self._session.post(uri, data=json.dumps(data))
 
+    def get_single_archive(self, archive_guid):
+        """Gets single archive information by GUID.
+        `REST Documentation <https://console.us.code42.com/apidocviewer/#Archive-get>`__
+
+        Args:
+            archive_guid (int): The GUID for the archive.
+
+        Returns:
+            :class:`py42.response.Py42Response`: A response containing archive information.
+        """
+        uri = u"/api/Archive/{}".format(archive_guid)
+        return self._session.get(uri)
+
+    def get_page(self, page_num, page_size=None, **kwargs):
+        """Gets an individual page of archives.
+        `REST Documentation <https://console.us.code42.com/apidocviewer/#Archive-get>`__
+
+        Args:
+            page_num (int): The page number to request.
+            page_size (int, optional): The number of archives to return per page.
+
+        Returns:
+            :class:`py42.response.Py42Response`
+        """
+        uri = u"/api/Archive"
+        page_size = page_size or settings.items_per_page
+        params = dict(
+            pgNum=page_num,
+            pgSize=page_size,
+            **kwargs
+        )
+        return self._session.get(uri, params=params)
+
+    def get_archives_from_value(self, id_value, id_type):
```

Comment:
If this is returning a generator, it needs to follow our standard of being prefixes with `get_all_`.

Perhaps `get_all_archives_from_value()`.

Reply:
Updated.